__tests__/app.test.js

```diff
@@ -192,3 +192,53 @@ describe("GET /api/articles, gets all articles", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("vote up an article", () => {
+    const newVotes = { inc_votes: 10 };
+
+    return request(app)
+      .patch("/api/articles/3")
+      .send(newVotes)
+      .expect(200) //its 200 bc we return
+      .then((response) => {
+        console.log(response.body, "res in TEST");
```

Comment:
remove console.logs before committing

Reply:
You still haven't removed the console.log